packages/react/src/Image/Image.test.tsx

```diff
@@ -0,0 +1,141 @@
+import React from 'react'
+import {render, cleanup} from '@testing-library/react'
+import '@testing-library/jest-dom'
+import {axe, toHaveNoViolations} from 'jest-axe'
+
+import {Image} from './'
+
+expect.extend(toHaveNoViolations)
+
+describe('Image', () => {
+  afterEach(cleanup)
+
+  it('no a11y violations', async () => {
+    const {container} = render(<Image src="https://source.unsplash.com/random" alt="" aspectRatio={undefined} />)
+    const results = await axe(container)
+
+    expect(results).toHaveNoViolations()
+  })
+
+  it('should pass a custom class to the image when provided', async () => {
+    const testClass = 'test'
+
+    const {container} = render(
+      <Image src="https://source.unsplash.com/random" alt="" className={testClass} aspectRatio={undefined} />
+    )
+
+    expect(container.querySelector(`img.${testClass}`)).toBeInTheDocument()
+  })
+
+  it('should return only an img element if no aspectRatio is provided and as is not picture', async () => {
+    const {container} = render(<Image src="https://source.unsplash.com/random" alt="" aspectRatio={undefined} />)
+
+    expect(container.querySelector('.Image-container')).not.toBeInTheDocument()
```

Comment:
Rather than querySelector, could we use either `getByAltText`, which serves dual advantage of checking for alt text, or use `getByRole('img')`?